sources/Metal/iTermMetalDriver.m

```diff
@@ -970,6 +971,7 @@ - (void)updateCursorGuideRendererForFrameData:(iTermMetalFrameData *)frameData {
     }
     [_cursorGuideRenderer setColor:frameData.perFrameState.cursorGuideColor];
     _cursorGuideRenderer.enabled = frameData.perFrameState.cursorGuideEnabled;
+    _cursorGuideRenderer.venabled = frameData.perFrameState.cursorVGuideEnabled;
```

Comment:
Note that when breaking enabled up into horizontal and vertical that the `enabled` property affects the renderer as as a whole, so it should be on when at least one of the guides is enabled. Add a new horizontalEnabled property to complement the verticalEnabled property.

Reply:
These have been separated.  The relevant top-level (ie whole-module) check of _enabled has been re-written to check both.